requirements.txt

```diff
@@ -22,4 +22,6 @@ httpx
 pydantic
 websockets>=10.0
 typing_extensions
-aiofiles
\ No newline at end of file
+aiofiles
+huggingface_hub
```

Comment:
I thought huggingface_hub was already added for the new analytics?

The new semantic_version dependency makes finding a right semver match a one liner.